packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -2081,41 +2349,52 @@ export interface GeometryOptions {
  * If this type is not supported, an error will be raised.
  *
  * @example <caption>Defining a Geometry type attribute</caption>
```

Comment:
![image](https://user-images.githubusercontent.com/13023439/212564562-5ab36dc4-391c-49bf-a1c6-6f4ca3fa109b.png)
We're having different examples here, but apart from that it still renders incorrectly